Data-Standards/Patch-Seq/electrophysiology-metadata-schema.json

```diff
@@ -0,0 +1,145 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "electrophysiology-metadata-schema.json",
+  "title": "Intracellular electrophysiology metadata",
+  "description": "Formal description of the IC Ephys Metadata List",
+  "author": "Ben Dichter",
+  "contact": "ben.dichter@gmail.com",
+  "version": "0.1.0",
+  "type": "object",
+  "required": [
+    "species",
+    "subject id",
+    "subject age",
+    "subject sex",
+    "subject date of birth",
+    "citation policy",
+    "institution",
+    "genotype",
+    "cre line",
+    "external solution recipe",
+    "recording temperature",
+    "electrode resistance",
+    "electrode internal solution recipe",
+    "specimen id",
+    "electrode id",
+    "stimulus electrode id",
+    "stimulus id",
+    "stimulus type name"
+    ],
+  "properties": {
+    "species": {"type": "string"},
+    "subject id": {"type": "string"},
+    "subject age": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "string",
+          "description": "Using ISO 8601 duration"
```

Comment:
How does this handle ages reported as `P14`?

Reply:
Does P14 mean 14 days after birth? Then age in ISO 8106 would be `P14D`.